es-ds-docs/pages/molecules/modal.vue

```diff
@@ -106,6 +112,42 @@ onMounted(async () => {
                     </es-button>
                 </template>
             </es-modal>
+
+            <es-modal
+                id="no-header-modal"
+                :visible="noHeaderModalVisible"
+                hide-header
+                size="lg"
+                @hidden="noHeaderModalVisible = false">
+                <template #modal-title> Arcadia Reviews </template>
```

Comment:
And then once the above setup is complete, removing this should remove the header.